### ISSUE TITLE
Switch statement on address update in case of $null

### ISSUE THREAD

**Issue by danijeljw** [oseyx/SUMS-Mass-Address-Update#3]
Using something like this:
```powershell
$carrot = 'cabbage'
Switch($carrot)
{
    $null
    {
    }
    default
    {
        "hello"
    }
}
```
It can be inferred that each change on the user account address can reflect _ONLY_ if there is data to match.

Switch has no default opposite to `$null` so this will ensure the data matches and applies.

Error comes up like this otherwise:
```powershell
Set-ADUser : replace
At G:\SUMS-Mass-Address-Update\main.ps1:22 char:13
+             Set-ADUser -Identity $_ -StreetAddress $Street -City $Cit ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (paula.davis:ADUser) [Set-ADUser], ADInvalidOperationException
    + FullyQualifiedErrorId : ActiveDirectoryServer:0,Microsoft.ActiveDirectory.Management.Commands.SetADUser
```

**Comment by danijeljw** [comment]
Changes did not rectify this.

Will pivot on the `$Country` field where **NZ** has no states. To invalidate the option.

**Comment by danijeljw** [comment]
Commit 8791c41 rectifies the issue. By pivoting on the `$Country` field for **AU** or **NZ** we can then isolate the `$State`.

In some instances, there are other fields that are blank and rather than working out what is the problem, it was easier to use the `-Length` module and determine if the `[string]` was longer than _0_. This enables us to act on it using a developed **Switch** statement in place for each line of the record.

**Comment by danijeljw** [comment]
Resolved by pull request c6d3f3a
